### ISSUE TITLE
Writer should record number of events and bytes unacknowledged

### ISSUE THREAD

**Issue by claudiofahey** [pravega/pravega-benchmark#56]

**Comment by kmgowda** [comment]
This have shared int implementation for multi thread cause the sync issues and in turn reduces the strength of tool to pump more data to pravega client.

**Comment by claudiofahey** [comment]
FYI I have a branch that implements this.
I will create a PR shortly.
https://github.com/claudiofahey/pravega-benchmark/tree/issue-56-track-unacked

**Comment by kmgowda** [comment]
include me @kmgowda  as reviewer

**Comment by claudiofahey** [comment]
I created PR #59. I do not have access to add you as a reviewer. Can you add me to this repo?

**Comment by RaulGracia** [comment]
@claudiofahey are you still interested on submitting the PR for this issue?

**Comment by claudiofahey** [comment]
Yes but I don't know when I'll have time to finish this. If this is needed soon, it would be better for somebody else to pick this up. My original solution to this is at https://github.com/pravega/pravega-benchmark/pull/59/commits/163da484239b40868da762c228d3f2c2db29cbdb.
